config/mvi_schema/mock_mvi_responses.yml.example

```diff
@@ -84,9 +84,20 @@ find_candidate:
     given_names: ['JERRY', 'M']
     icn: '1008709435V263289^NI^200M^USVHA^P'
     mhv_id: '12210827^PI^200MH^USVHA^A'
-    vba_corp_id: ~
+    vba_corp_id: '600036156'
```

Comment:
actually ya that's fine, carets postfix is for identifying the id in the real service, but for mock it'll work, so either way